src/sentry/sentry_metrics/querying/metadata/correlations.py

```diff
@@ -458,8 +500,8 @@ def _get_segments_aggregates_query(
             Column("segment_name"),
             Column("profile_id"),
         ],
-        # For now, we order by descending duration.
-        orderby=[OrderBy(Column("duration"), Direction.DESC)],
+        # For now, we order by descending segment duration.
+        orderby=[OrderBy(Column("segment_duration"), Direction.DESC)],
```

Comment:
I have fixed this, since it was causing issues because the dbms was not able to understand if `duration` was the `duration` column or the `duration` aggregate.